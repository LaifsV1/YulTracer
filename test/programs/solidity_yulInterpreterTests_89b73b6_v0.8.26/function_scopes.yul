{
    f(1)
    function f(i) {
        if i { g(1) }
        function g(j) {
            if j { h() }
            f(0)
            function h() {
                g(0)
            }
        }
        sstore(i, add(i, 7))
  }

  // Added storage dump:
  printhex(sload(0x0))
  printhex(sload(0x1))
}
// ----
// Trace:
// Memory dump:
// Storage dump:
//   0000000000000000000000000000000000000000000000000000000000000000: 0000000000000000000000000000000000000000000000000000000000000007
//   0000000000000000000000000000000000000000000000000000000000000001: 0000000000000000000000000000000000000000000000000000000000000008
// Transient storage dump:
