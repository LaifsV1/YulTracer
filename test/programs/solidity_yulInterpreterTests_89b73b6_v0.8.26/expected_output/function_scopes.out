0000000000000000000000000000000000000000000000000000000000000007
0000000000000000000000000000000000000000000000000000000000000008
